pkg/operation/common/types.go

```diff
@@ -485,6 +505,9 @@ const (
 
 	// CertBrokerResourceName is the name of the Cert-Broker resources.
 	CertBrokerResourceName = "cert-broker"
+
+	// KubectlVersion specifies the kubectl version at shoot clusters node
```

Comment:
Better description: `KubectlVersion specifies the version of kubectl that is used by the scripts bootstrapping shoot worker nodes, e.g. the cloud-config-downloader script.`